invenio_users_resources/administration/views/users.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio administration users view module."""
+from flask import current_app
+from invenio_administration.views.base import (
+    AdminResourceDetailView,
+    AdminResourceListView,
+)
+from invenio_i18n import lazy_gettext as _
+
+
+class UsersListView(AdminResourceListView):
+    """Configuration for users sets list view."""
+
+    api_endpoint = "/users"
+    name = "users"
+    resource_config = "users_resource"
+    title = "User management"
+    menu_label = "Users"
+    category = "User management"
+    pid_path = "id"
+    icon = "exchange"
```

Comment:
```suggestion
    icon = "users"
```

The value here is used for [React-Semantic-UI icons](https://react.semantic-ui.com/elements/icon/); probably a fitting icon would be the "users" icon here?